pkg/stub/handler.go

```diff
@@ -887,12 +932,23 @@ func (h *Handler) createSamples(cfg *v1.Config, updateIfPresent, registryChanged
 
 		err = h.upsertTemplate(template, t, cfg)
 		if err != nil {
+			if kerrors.IsAlreadyExists(err) {
+				continue
+			}
```

Comment:
Add comment (swallowed error)

Reply:
actually now that I've sorted out which informer api to use, and am no longer seeing odd timing windows on deletes/recreates with stale caches, we should not need this anymore ... reverting this addition instead.